src/features/campaigns/components/CampaignDetailsForm.tsx

```diff
@@ -22,16 +20,9 @@ const CampaignDetailsForm = ({
   onCancel,
   campaign,
 }: CampaignDetailsFormProps): JSX.Element => {
-  const { orgId } = useNumericRouteParams();
   const messages = useMessages(messageIds);
 
-  const membershipsFuture = useMemberships();
-  const activeMembership = membershipsFuture.data?.find(
-    (m) => m.organization.id == orgId
-  );
-  const userProfile = activeMembership?.profile;
-
-  const [selectedManager, setSelectedManager] = useState<ZetkinPerson | null>(
+  const [selectedManager] = useState<ZetkinPerson | null>(
```

Comment:
I believe you can also delete this (and all uses of it). Do you agree?

Reply:
Sure ill come around to it soon.